src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -20,6 +21,12 @@ public ValueStringBuilder(Span<char> initialBuffer)
             _chars = initialBuffer;
             _pos = 0;
         }
+        public ValueStringBuilder(int initialCapacity)
```

Comment:
Can you please add some test(s) to ValueStringBuilderTests for your new functionality?